atlp-rwanda/cod-be
# #181414543 Configure BN Postgres Database & Sequelize ORM

#### What does this PR do?
1. It Configures Postgres Database & Sequelize ORM of Barefoot Nomad

#### Description of Task to be completed?

1. Set up the migrations and models structure.
2. Have a dummy user table and model setup to test the whole configuration.
3. Add the various required setup in the README if any is required.

#### How should this be manually tested?

1. Open the `src/database` directory  
2. Check the `src/database/config` directory to check database config file
3. Check the `src/database/migrations` directory to check initial Migration File
4. Check the `src/database/models` directory to check models
5. Check the `src/database/seeders` directory to check seeders

#### What are the relevant pivotal tracker stories?
Pivotal Tracker Story: [https://www.pivotaltracker.com/story/show/181414543](url)
